.github/prlint.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": [
+    {
+      "pattern": "^(feature|fix|issue|bug|docs|cicd|refactor|test):\\s",
+      "message": "Your title needs to be prefixed with a topic. Allowed prefixes: feature,fix,issue,bug,docs,cicd,refactor,test"
+    }
+  ],
+ "body": [
+    {
+      "pattern": ".{1,}",
+      "message": "You need literally anything in your description"
+    }
+  ],
+  "assignee.login": [
```

Comment:
I'm not sure about the assigned is required. We don't have a component owner and people don't know who can review PR.

Reply:
Ok, fixed below